subsys/bluetooth/host/hci_core.c

```diff
@@ -2502,6 +2502,41 @@ void bt_hci_le_transmit_power_report(struct net_buf *buf)
 }
 #endif /* CONFIG_BT_TRANSMIT_POWER_CONTROL */
 
+#if defined(CONFIG_BT_PATH_LOSS_MONITORING)
+void bt_hci_le_path_loss_threshold_event(struct net_buf *buf)
+{
+	struct bt_hci_evt_le_path_loss_threshold *evt;
+	struct bt_conn_le_path_loss_threshold_report report;
+	struct bt_conn *conn;
+
+	evt = net_buf_pull_mem(buf, sizeof(*evt));
+
+	if (evt->zone_entered > BT_CONN_LE_PATH_LOSS_ZONE_ENTERED_HIGH) {
+		LOG_ERR("Invalid zone in bt_hci_evt_le_path_loss_threshold");
```

Comment:
```suggestion
		LOG_ERR("Invalid zone %u in bt_hci_evt_le_path_loss_threshold", evt->zone_entered);
```

For debugging purposes it's a good idea to also log the invalid value :)